Public/Src/Engine/Processes/SharedOpaqueOutputHelper.cs

```diff
@@ -33,6 +33,16 @@ private static class Win
             /// <exception cref="BuildXLException">When the timestamp cannot be set</exception>
             public static void SetPathAsSharedOpaqueOutput(string expandedPath)
             {
+                // In the case of a no replay, this case can happen if the file got into the cache as a static output,

```

Comment:
just reverting this to how it originally was: call it only here for Windows instead of calling it uniformly regardless of the current platform.